examples/seismic/acoustic/operators.py

```diff
@@ -54,13 +67,19 @@ def iso_stencil(field, m, s, damp, kernel, **kwargs):
 def ForwardOperator(model, geometry, space_order=4,
                     save=False, kernel='OT2', **kwargs):
     """
-    Constructor method for the forward modelling operator in an acoustic media
-
-    :param model: :class:`Model` object containing the physical parameters
-    :param source: :class:`PointData` object containing the source geometry
-    :param receiver: :class:`PointData` object containing the acquisition geometry
-    :param space_order: Space discretization order
-    :param save: Saving flag, True saves all time steps, False only the three
+    Construct a forward modelling operator in an acoustic media.
+
+    Parameters
+    ----------
+    model : Model
+        Object containing the physical parameters.
+    geometry : AcquisitionGeometry
+        Geometry object that contains the source (SpareTimeFunction) and
+        receivers (SparseTimeFunction) and their position.
+    space_order : int, optional
+        Space discretization order.
+    save : int or Buffer, optional
+        Saving flag, True saves all time steps. False only the three. Defaults to False.
```

Comment:
"False only the three" is really odd english. People won't understand what's going on, but we can leave this fix for another day...